cypress/page/address-step.page.ts

```diff
@@ -2,7 +2,7 @@ class AddressStepPage {
   private proceedToCheckoutBtn: string;
 
   constructor() {
-    this.proceedToCheckoutBtn = ".cart_navigation button";
+    this.proceedToCheckoutBtn = ".cart_navigation .button";
```

Comment:
see if you can use an attribute of a tag to indicate that we are looking for the proceed to checkout button